MahjongAI/Constants.cs

```diff
@@ -1,25 +1,24 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using MahjongAI.Models;
 
 namespace MahjongAI
 {
     static class Constants
     {
-        public static readonly IReadOnlyDictionary<MajsoulRegion, string> MAJSOUL_API_URL_PRIFIX = new Dictionary<MajsoulRegion, string>()
+        private const string CN_GAME_SERVER_HOST = "https://game.maj-soul.com/1";
```

Comment:
You can make this public so that you don't have to put it in the dictionary below.